openedx/core/djangoapps/dark_lang/middleware.py

```diff
@@ -8,14 +8,14 @@
 the SessionMiddleware.
 """
 from django.conf import settings
+from django.utils.translation.trans_real import parse_accept_lang_header
+from django.utils.translation import LANGUAGE_SESSION_KEY
 
-from dark_lang import DARK_LANGUAGE_KEY
-from dark_lang.models import DarkLangConfig
+from openedx.core.djangoapps.dark_lang import DARK_LANGUAGE_KEY
+from openedx.core.djangoapps.dark_lang.models import DarkLangConfig
```

Comment:
`from dark_lang import DarkLang`, so now we have three casing/spacing styles for this one construct :)